code_schemes/s01e01.json

```diff
@@ -1,8 +1,112 @@
   "Version": "0.0.0.1",
   "Codes": [
+    {
+        "CodeID": "code-95f5ec39",
+        "CodeType": "Normal",
+        "DisplayText": "discrimination",
+        "NumericValue": 0,
+        "StringValue": "discrimination",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-e5c090d6",
+        "CodeType": "Normal",
+        "DisplayText": "need for cultural change",
+        "NumericValue": 1,
+        "StringValue": "need_for_cultural_change",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-f1d76a3c",
+        "CodeType": "Normal",
+        "DisplayText": "nepotism",
+        "NumericValue": 2,
+        "StringValue": "nepotism",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-0a5bc746",
+        "CodeType": "Normal",
+        "DisplayText": "such issues exist",
+        "NumericValue": 3,
+        "StringValue": "such_issues_exist",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-aff98330",
+        "CodeType": "Normal",
+        "DisplayText": "clannism",
+        "NumericValue": 4,
+        "StringValue": "clannism",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-7a79977b",
+        "CodeType": "Normal",
+        "DisplayText": "gender discrimination",
+        "NumericValue": 5,
+        "StringValue": "gender_discrimination",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-fb1c06d5",
+        "CodeType": "Normal",
+        "DisplayText": "women exclusion from decision making",
+        "NumericValue": 6,
+        "StringValue": "women_exclusion_from_decision_making",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-68ab8a5b",
+        "CodeType": "Normal",
+        "DisplayText": "youth unemployment",
+        "NumericValue": 7,
+        "StringValue": "youth_unemployment",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-883a7932",
+        "CodeType": "Normal",
+        "DisplayText": "youths voices not heard",
+        "NumericValue": 8,
+        "StringValue": "youths_voices_not_heard",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-f97c6d75",
+        "CodeType": "Normal",
+        "DisplayText": "demoralization",
+        "NumericValue": 9,
+        "StringValue": "demoralization",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-a4a0f10b",
+        "CodeType": "Normal",
+        "DisplayText": "illegal emigration ",
+        "NumericValue": 10,
+        "StringValue": "illegal_emigration ",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-13ca3fdf",
+        "CodeType": "Normal",
+        "DisplayText": "other",
+        "NumericValue": 11,
+        "StringValue": "other",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-3e46d3d7",
+        "CodeType": "Normal",
+        "DisplayText": "positive highlight",
```

Comment:
What does this code mean?

Reply:
It covers a couple of responses from people appreciating the show and its content. Reference the sub themes here. https://docs.google.com/spreadsheets/d/1YliX-AWnKiPu9dKCPx_t8-DzSojorMcPXQ6UsqTfmKI/edit?usp=sharing